packages/components/cascader/src/cascader.vue

```diff
@@ -60,10 +72,11 @@
             <el-icon
               v-else
               key="arrow-down"
-              :class="cascaderIconKls"
+              :class="[cascaderIconKls]"
```

Comment:
**Question:** why add this bracket though? Since this is already an array why would we do this?